docs/manual/src/udl/namespace.md

```diff
@@ -8,7 +10,10 @@ namespace math {
 };
 ```
 
-It serves multiple purposes:
-- It identifies the name of the generated Rust scaffolding file `<namespace>.uniffi.rs`.
-- It identifies the package name of the generated foreign-language bindings (e.g. `uniffi.<namespace>` in Kotlin)
-- It also contains all [top-level *functions*](./functions.md) that get exposed to foreign-language bindings.
+which might be used as `from math import exp`/`import math`/`import omg.wtf.math.exp` etc.
+
+Proc-macros use the crate name as the namespace by default, but it can be specified with
```

Comment:
heh, and here with a dash. We should at least be consistent.

Reply:
yeah, thanks!